spec/sync-settings-spec.coffee

```diff
@@ -145,15 +145,16 @@ describe "SyncSettings", ->
           , (err, res) ->
             expect(res.files['styles.less']).not.toBeDefined()
 
-      it "back up the user init.coffee file", ->
+      it "back up the user Init Script file", ->
         atom.config.set('sync-settings.syncInit', true)
+        initPath = atom.getUserInitScriptPath()
         run (cb) ->
           SyncSettings.backup cb
         , ->
           run (cb) =>
             SyncSettings.createClient().gists.get({id: @gistId}, cb)
           , (err, res) ->
-            expect(res.files['init.coffee']).toBeDefined()
+            expect(res.files[path.basename(initPath)]).toBeDefined()
```

Comment:
Please keep the changes for the `init` file in a separate PR since they are unrelated to the config path problem addressed in this PR.

Reply:
OK, fixed. 

But in my environment, both init.coffee and init.js exists. 

```
      it "back up the user init.coffee file", ->
        atom.config.set('sync-settings.syncInit', true)
        run (cb) ->
          SyncSettings.backup cb
```

this test backups init.js instead of init.coffee, so my test fails locally.

Reply:
The patch for the init files is still valid. Please create a separate PR for it though. Then they can get merged in whatever order is necessary to keep CI passing.

Reply:
OK, i will create.